src/main/scala/org/broadinstitute/hail/driver/IBDCommand.scala

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.hail.driver
+
+import org.broadinstitute.hail.annotations.Annotation
+import org.broadinstitute.hail.expr.{BaseAggregable, EvalContext, Parser, TBoolean, TDouble, TGenotype, TVariant}
+import org.broadinstitute.hail.methods.IBD
+import org.broadinstitute.hail.variant.{Variant, VariantDataset}
+import org.kohsuke.args4j.{Option => Args4jOption}
+
+object IBDCommand extends Command {
+
+  class Options extends BaseOptions {
+    @Args4jOption(required = false, name = "-m", aliases = Array("--minor-allele-frequency"),
+      usage = "An expression for the minor allele frequency of the current variant, `v', given the variant annotations `va'")
+    var computeMafExpr: String = _
+
+    @Args4jOption(required = false, name = "--unbounded",
+      usage = "Allows the estimations for Z0, Z1, Z2, and PI_HAT to take on biologically-nonsense values.")
```

Comment:
Z0, Z1, and Z2 are supposed to sum to one. IIRC, I lifted this line from the PLINK paper. I'll add something.